src/Character.js

```diff
@@ -500,13 +500,15 @@ class Character extends Metadatable(EventEmitter) {
    * @fires Character#unfollowed
    */
   unfollow() {
-    this.following.removeFollower(this);
-    /**
-     * @event Character#unfollowed
-     * @param {Character} following
-     */
-    this.emit('unfollowed', this.following);
-    this.following = null;
+    if (this.following) {
```

Comment:
I'm a proponent of early return. So instead of wrapping the entire function body in an if, you can instead just do

```js
if (!this.following) {
  return
}
```